components/datatypes/webpagedetails.example.2.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I would change the example to make it big more clear how this will be actually used. 
```javascript
{
  "xdm:siteSection": "Product Details Page",
  "xdm:server": "example.com",
  "xdm:name": "Scottish Haggis Product Details",
  "xdm:viewName": "FAQ Tab",
  "xdm:URL": "https://www.example.com",
  "xdm:errorPage": false,
  "xdm:homePage": true,
  "xdm:pageViews": {
    "xdm:value": 1.0
  }
}
```

Reply:
thank you Justin, will add more detailed example